doc/changes/latest.inc

```diff
@@ -19,6 +19,10 @@ Current (0.23.dev0)
 
 .. |New Contributor| replace:: **New Contributor**
 
+.. |Jack Zhang| replace:: **Jack Zhang**
+
+.. |New Contributor| replace:: **New Contributor**
```

Comment:
now 


.. |New Contributor| replace:: **New Contributor**

is written twice

Reply:
Fixed.